openedx/features/job_board/forms.py

```diff
@@ -16,3 +16,13 @@ class Meta:
             'compensation': RadioSelect,
             'hours': RadioSelect,
         }
+        labels = {
+            'city': 'City:',
+            'country': 'Country:',
+            'application_link': 'APPLICATION LINK:',
+            'website_link': 'WEBSITE LINK:',
```

Comment:
We also provided verbose_name for `application_link` and `website_link`, but there these aren't all in capital case. How about removing them from here and update them in model and makemigrations. or may be we can make them uppercase using CSS `text-transform:uppercase;` and keep the verbose_name in model same.

Reply:
See https://github.com/philanthropy-u/edx-platform/pull/1669#discussion_r455667586